### PR TITLE
Fixed bug ehere start tour param was not sent when restarting touyr

### LINKED ISSUE #2865: Bug: Create a board from Welcome board template should show tour points
## Steps to reproduce the behavior

1. Create a user and open Boards
2. Decline the tour 
3. Go to template picker
4. Select  welcome board template 
5. Create a board from welcome board (or select Product Tour from settings)
6. It does not show the tour poins

## Expected behavior
Create a board from the welcome board templates should show the tour points

## Screenshots (optional)
Screenshot 2022-04-21 at 1 28 57 PM

## Edition and Platform
 - Edition: Mattermost Boards (plugin)
 - Version: v0.16.0
 - Browser and OS: Chrome on Mac

## Additional context (optional)

Sev 3

### PR DESCRIPTION
**Pull request by harshilsharma63**
Fixes #2865

The bug was that I wasn't setting the user prop that indicated the tour has started. So skipping from the welcome page never allowed you to restart the tour.

### CODE REVIEW

File: webapp/src/components/boardTemplateSelector/boardTemplateSelector.test.tsx [hunk 1]
```diff
@@ -64,6 +69,7 @@ describe('components/boardTemplateSelector/boardTemplateSelector', () => {
     beforeAll(mockDOM)
     beforeEach(() => {
         jest.clearAllMocks()
+        // mockedOctoClient.getAllBlocks.mockResolvedValue([])
```
Comment (sbishel) [inline webapp/src/components/boardTemplateSelector/boardTemplateSelector.test.tsx:72]: nit: remove commented line.

File: webapp/src/components/globalHeader/globalHeaderSettingsMenu.test.tsx [hunk 2]
```diff
@@ -16,10 +16,15 @@ import {wrapIntl} from '../../testUtils'
 
 import TelemetryClient, {TelemetryCategory, TelemetryActions} from '../../telemetry/telemetryClient'
 
+import client from '../../octoClient'
+
 import GlobalHeaderSettingsMenu from './globalHeaderSettingsMenu'
 
+
```
Comment (sbishel) [inline webapp/src/components/globalHeader/globalHeaderSettingsMenu.test.tsx:23]: nit: unnecessary line

File: webapp/src/components/boardTemplateSelector/boardTemplateSelector.test.tsx [hunk 3]
```diff
@@ -18,8 +18,11 @@ import {Team} from '../../store/teams'
 import {IUser} from '../../user'
 import {mockDOM, mockStateStore, wrapDNDIntl} from '../../testUtils'
 
+import client from '../../octoClient'
+
 import BoardTemplateSelector from './boardTemplateSelector'
 
+
```
Comment (sbishel) [inline webapp/src/components/boardTemplateSelector/boardTemplateSelector.test.tsx:25]: nit: unnecessary line

**Review by sbishel — APPROVED** [review]
A couple nits, otherwise LGTM

File: webapp/src/components/boardTemplateSelector/boardTemplateSelector.test.tsx (hunk 3 above)
Comment (harshilsharma63) [reply webapp/src/components/boardTemplateSelector/boardTemplateSelector.test.tsx]: Done.

File: webapp/src/components/globalHeader/globalHeaderSettingsMenu.test.tsx (hunk 2 above)
Comment (harshilsharma63) [reply webapp/src/components/globalHeader/globalHeaderSettingsMenu.test.tsx]: Done.

File: webapp/src/components/boardTemplateSelector/boardTemplateSelector.test.tsx (hunk 1 above)
Comment (harshilsharma63) [reply webapp/src/components/boardTemplateSelector/boardTemplateSelector.test.tsx]: Done.

**Pull request merged**